FO-DICOM.Core/Network/DesktopNetworkListener.cs

```
@@ -88,8 +105,29 @@ public Task StartAsync()
 
                 return null;
             }
-            catch
+            catch (OperationCanceledException)
+            {
+                if (logger.IsEnabled(LogLevel.Debug))
+                {
+                    logger.LogDebug("Listener for {IPAddress}:{Port} has stopped because it was cancelled",
+                        _endpoint.Address.ToString(), _endpoint.Port);
+                }
+
+                return null;
+            }
+            catch (SocketException ex) when (ex.SocketErrorCode == SocketError.OperationAborted)
+            {
+                if (logger.IsEnabled(LogLevel.Debug))
+                {
+                    logger.LogDebug("Listener for {IPAddress}:{Port} has stopped because the connection was closed",
+                        _endpoint.Address.ToString(), _endpoint.Port);
+                }
+                return null;
+            }
+            catch(Exception exception)
             {
+                logger.LogError(exception, "An error occurred while listening for inbound client connections to {IPAddress}:{Port}", 
```

This line was the main reason I created this pull request.
Today, errors that occur here are silently swallowed.